en/api/internals-builder.md

```diff
@@ -19,7 +19,8 @@ this.nuxt.hook('build:done', (builder) => {
 Hook                 | Arguments                                  | When
 ---------------------|--------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------
 `build:before`       | (nuxt, buildOptions)                       | Before Nuxt build started
-`build:prepared`     | (nuxt, buildOptions)                       | The build directories have been created
+`builder:prepared`     | (nuxt, buildOptions)                       | The build directories have been created
```

Comment:
I saw that actual hook [here](https://github.com/nuxt/nuxt.js/blob/dev/packages/builder/src/builder.js#L155) is `builder:prepared` instead of `build:prepared`

Reply:
Nice catch @starikovm